packages/devlow-bench/src/describe.ts

```diff
@@ -131,14 +159,14 @@ export function reportMeasurement(
     if (!prev) {
       throw new Error(`No measurement named ${relativeTo} found`);
     }
-    if (prev.unit !== "ms") {
+    if (prev.unit !== unit) {
       throw new Error(
         `Measurement ${relativeTo} is not a "${unit}" measurement`
       );
     }
     reportedValue -= prev.value;
   }
-  currentScenario.iface.measurement(
+  await currentScenario.iface.measurement(
```

Comment:
`measurement` is actually async and to handle errors correctly we need to await it